api/api-doc/storage_service.json

```diff
@@ -104,6 +104,54 @@
             }
          ]
       },
+      {
+         "path":"/storage_service/toppartitions/",
+         "operations":[
+            {
+               "method":"GET",
+               "summary":"Toppartitions query",
+               "type":"toppartitions_query_results",
+               "nickname":"toppartitions_generic",
+               "produces":[
+                  "application/json"
+               ],
+               "parameters":[
+                  {
+                     "name":"name_list",
+                     "description":"Optional list of column family names in keyspace:name format, narrowing down the search. Omitting the name (e.g. 'ks:') will match against all families in the keyspace",
+                     "required":false,
+                     "allowMultiple":true,
```

Comment:
we are currently don't support multiple parameters in the query

Reply:
In what way do we not support it?
Should I change it explicitly to a string array?

Reply:
in the way that if you add multiple parameters with the same name you will only get the last.
What you will get is a string, this is why we encode it as a comma delimited.

Reply:
ah, ok
then an array it is